tendrl/ceph_integration/manager/rpc.py

```diff
@@ -20,13 +20,14 @@
 
 class EtcdRPC(object):
 
-    def __init__(self, manager):
+    def __init__(self, manager, Etcdthread):
```

Comment:
EtcdRPC  have infinite loop. It will never stop. We have this check in other repos also.